### ISSUE TITLE
Reroute google voice sms/mms to/from different sms app?

### ISSUE THREAD

**Issue by markreg** [kmark/XHangouts#18]
Is it possible to implement a reroute feature for incoming/outgoing google voice sms/mms messages to send/receive in a different sms app instead of hangouts? Kind of like XVoice+ but ultimately routing the messages through Hangouts I suppose.

The Hangouts app's interface is a little less convenient than the regular Messenger app but since I use Google Voice I've been stuck with the Hangouts app.

Would love to hear your thoughts on this.

**Comment by kmark** [comment]
So this would effectively do the same thing (on the surface) as XVoicePlus, correct? I'm guessing that no longer works or is no longer updated?

**Comment by markreg** [comment]
Correct, XVoicePlus's latest release was over two years ago and I believe it only supported SMS at the time since the Google Voice app didn't support MMS and the Hangouts app didn't exist.
